flow-typed/Collections.js

```diff
@@ -14,14 +14,21 @@ declare type CollectionState = {
   pending: CollectionGroup,
   edited: CollectionGroup,
   builtin: CollectionGroup,
-  saved: Array<string>,
   isResolvingCollectionById: { [string]: boolean },
```

Comment:
--- non blocker ---
<img width="248" alt="image" src="https://user-images.githubusercontent.com/64950861/173492912-93867a71-139f-483c-b11e-2b68b9108842.png">

probably not meant to wrap